doc/source/guide/spatial-partitioning.ipynb

```diff
@@ -0,0 +1,717 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "aca93ab3-f452-4725-b864-b41cd0327954",
+   "metadata": {},
+   "source": [
+    "# Spatial partitioning in Dask-GeoPandas\n",
+    "\n",
+    "Dask-GeoPandas has implemented `spatial_shuffle` method to repartition `Dask.GeoDataFrames` geographically.\n",
+    "For those who are not familiar with Dask, a Dask DataFrame is internally split into many partitions, where each partition is a Pandas DataFrame.\n",
+    "Partitions are split vertically along the index; \n",
+    "\n",
+    "<img src=\"https://docs.dask.org/en/latest/_images/dask-dataframe.svg\" alt=\"drawing\" width=\"300\"/>\n",
+    "\n",
+    "There are numerous strategies that can be used to partition Dask DataFrames, which determine how the elements of a DataFrame are separated into each resulting partition.\n",
+    "Common strategies to partition Dask DataFrames include using a fixed number of partitions or partitions based on column values.\n",
+    "Dask-GeoPandas partitions GeoDataFrames by taking advantage of their spatial structure.\n",
```

Comment:
```suggestion
    "Dask-GeoPandas can partition GeoDataFrames by taking advantage of their spatial structure.\n",
```

(we don't necessarily do it, you need to explicitly do it, so "can" feels more correct)